### ISSUE TITLE
Improve performance of notifier task of new items

### ISSUE THREAD

**Issue by AlfredSK** [friendica/friendica#9453]
### Is the feature request related to a problem? Please describe.
I have observed that Friendica needs much more time to deliver new posts and comments than other systems like Mastodon, Pleroma or Misskey. When I send a post or comment with these other systems I get the content in nearly no time - it takes only 1-5 seconds until I receive the item on Friendica. But when I reply to such a post or create a new one myself it can take several minutes until that item has been delivered from Friendica to the other side.

The delay has nothing to do with the number of items in the workerqueue. The delay also exists when the queue is completely empty while creating the item.

What I observed is that the notifier task 'wall-new' and the 'create shadow entry' job is created and queued with prio 20. Then these jobs are waiting in the queue for being processed. That can take several minutes (probably waiting for the next cron interval). The whole waiting in the queue is completely useless. The notifier task itself should be processed *immediately* and should only be queued when something goes wrong or the system is overloaded.

For the user it looks like something is wrong because the item is showing an hourglass indicator for several minutes.

### Describe the feature you'd like
I would like to have a behavior where that initial task is processed immediately to speed up the delivery especially when the queue is nearly empty.

### Additional context
I'm using cron to run the worker.
Cron interval is set to 3 minutes.
2020.12-dev

PS: I saw the PR where the 'create shadow entry' has been removed. That's the reason why I remembered the issue. :-)

**Comment by annando** [comment]
The delivery isn't done solely done via the cron stuff. Immediately after the post is done the system should automatically fork a process to start the delivery. You could also use the daemon. Just - instead of using the cron - start `bin/daemon start` in the command line. This should perform the delivery very fast as well.

**Comment by AlfredSK** [comment]
> Immediately after the post is done the system should automatically fork a process to start the delivery.

Yeah. "Should". But it doesn't. Under circumstances unknown it takes several minutes until the delivery starts - even with an empty queue. 
The create notifier tasks doesn't get processed. It is just lingering in the workerqueue. After the notifier task has been created/processed the actual delivery jobs are processed pretty fast. But it takes too much time until the whole thing gets started.

**Comment by annando** [comment]
How is your "proc_open" setting in the admin interface?

**Comment by annando** [comment]
I just added some logging. This could help debugging if processes are spawned at all.

**Comment by AlfredSK** [comment]
'Don't use proc_open' is not enabled. A double negation again. ;-)

**Comment by AlfredSK** [comment]
Btw. we should consider unifying the wording of options. It should always be: enable an option to enable a function; disable an option to disable a function.

That 'enable option to disable a function' is confusing. Especially because we mix this all the time.

But that's another issue. :-)

**Comment by AlfredSK** [comment]
Hmm. Since that 'shadow entry' thing has been removed it seems to be much faster. For example I don't see that hourglass indicator anymore after submitting a comment. The comment item is showing the white paper plane instantly now. Needs more testing...

**Comment by AlfredSK** [comment]
...more testing:
Sometimes it's super fast and sometimes it still takes more than 1.5 minutes until the (AP) delivery starts.

This is how it looks during that delay:

![Screenshot_20201024-224008](https://user-images.githubusercontent.com/3656054/97093229-761dfd80-164a-11eb-8635-c38a5433fe81.png)
